src/sagemaker/clarify.py

```diff
@@ -971,7 +973,7 @@ def _run(
     def run_pre_training_bias(
         self,
         data_config,
-        data_bias_config,
+        bias_config,
```

Comment:
This change do not look backward compatible

Reply:
good point. let me revert it back